README.md

```diff
@@ -1,2 +1,24 @@
-# hw01
-Repository for homework 1
+# Biography
+
+
+![](https://media.licdn.com/mpr/mpr/shrinknp_400_400/AAEAAQAAAAAAAAqhAAAAJDNlZGE2ZDY3LTZlYTQtNGU1OS1iZTViLThiNjAzOWRlZmI4Yg.jpg)
+
+## Ying Zeng
+
+https://github.com/zyzyxzy
+
+
+### Introduction
+
+Ying Zeng is currently a **MAPSS** student in *psychology* at the University of Chicago. She graduated from **Peking University** in 2017, with dual bachelor degrees in *psychology* and *economics*.
+
+
+### Research Interests
+
+My research interests covers various topics in social psychology and consumer behavior. Specifically, I am quite interested in:
+
+* Judgement decision making
+* Perspective taking
```

Comment:
I don't know what 'perspective taking' is, but you've managed to use another required aspect in a coherent context.